src/conn/conn_dhandle.c

```diff
@@ -65,7 +65,7 @@ __conn_dhandle_config_set(WT_SESSION_IMPL *session)
      * in metaconf. If we fail before we copy a reference to it into the object's configuration
      * array, we must free it, after the copy, we don't want to free it.
      */
-    WT_ERR(__wt_calloc_def(session, 3, &dhandle->cfg));
```

Comment:
Okay this one was confusing me, we allocate an array of 3 entries then set the 4th entry to NULL?

Reply:
I should've scrolled up.